src/DonationForms/resources/registrars/templates/fields/Phone.tsx

```diff
@@ -1,15 +1,89 @@
-import InputMask from 'react-input-mask';
-
 import {PhoneProps} from '@givewp/forms/propTypes';
+import {useEffect} from 'react';
+import InputMask from 'react-input-mask';
 
-export default function Phone({Label, ErrorMessage, placeholder, description, phoneFormat, inputProps}: PhoneProps) {
+/**
+ * @unreleased
+ */
+export default function Phone({
+    Label,
+    ErrorMessage,
+    placeholder,
+    description,
+    phoneFormat,
+    inputProps,
+    fieldError,
+    intlTelInputSettings,
+}: PhoneProps) {
     const FieldDescription = window.givewp.form.templates.layouts.fieldDescription;
+    const {useFormContext} = window.givewp.form.hooks;
+    const {setValue, setError, trigger} = useFormContext();
+
+    const intlTelInputId = inputProps.name + '_intl_tel_input';
```

Comment:
Regarding my previous comment about Refs instead of IDs you should be able to do:

`const inputRef = useRef(null)`

then likely within useEffect:
`const const intlTelInputId = inputRef.current`

actually while I was typing this I found this codepen that shows an example of this approach with this library from the maintainer: https://codepen.io/jackocnr/pen/RwePbZg

Reply:
Resolved in 8226c6dc942a22ff20fd024c8e359827901e5b76 and 4a7c5b3945a68d416dcac2f52da5037c4ff9fc8c